coderd/database/queries/workspacebuildparameters.sql

```diff
@@ -14,3 +14,28 @@ FROM
     workspace_build_parameters
 WHERE
     workspace_build_id = $1;
+
+-- name: GetUserWorkspaceBuildParameters :many
+SELECT
+    sub.name,
+    sub.value,
+    sub.created_at
+FROM (
+    SELECT
+        wbp.name,
+        wbp.value,
+        wb.created_at,
+        ROW_NUMBER() OVER (PARTITION BY wbp.name ORDER BY wb.created_at DESC) as rn
+    FROM
+        workspace_build_parameters wbp
+    JOIN 
+        workspace_builds wb ON wb.id = wbp.workspace_build_id
+    JOIN
+        workspaces w ON w.id = wb.workspace_id
+    WHERE
+        w.owner_id = $1
+        AND wb.transition = 'start'
```

Comment:
You need to skip parameters marked as "ephemeral" in the `template_version_parameters`. These are more like "debug" parameters.